staging/src/k8s.io/kubectl/pkg/scheme/install.go

```diff
@@ -80,4 +81,5 @@ func init() {
 	utilruntime.Must(Scheme.SetVersionPriority(rbacv1.SchemeGroupVersion, rbacv1beta1.SchemeGroupVersion, rbacv1alpha1.SchemeGroupVersion))
 	utilruntime.Must(Scheme.SetVersionPriority(schedulingv1alpha1.SchemeGroupVersion))
 	utilruntime.Must(Scheme.SetVersionPriority(storagev1.SchemeGroupVersion, storagev1beta1.SchemeGroupVersion, storagev1alpha1.SchemeGroupVersion))
+	kuberc.Install(Scheme)
```

Comment:
I think an isolated scheme used just for loading kuberc files, with just the kuberc API groups registered would be better (and we could make it strict if we want to). You can construct that privately in the package where kuberc files are read

Reply:
Updated.